packages/syft/src/syft/client/domain_client.py

```diff
@@ -89,6 +91,119 @@ class DomainClient(SyftClient):
     def __repr__(self) -> str:
         return f"<DomainClient: {self.name}>"
 
+    def demo(self) -> tuple[DomainClient, DomainClient, DomainClient]:
```

Comment:
maybe client.demo("mnist") ?

Reply:
Love this as a future feature. Don't feel it's a blocker for the first dataset or two to be in the API.